modules/installation/pages/install-proxy-uyuni.adoc

```diff
@@ -0,0 +1,52 @@
+
+USE IT AS TEMPLATE FOR PROXY INSTALL
+
+
+[[install-proxy-uyuni]]
+= Install Uyuni Proxy with openSUSE
+
+{productname} Proxy can be installed on openSUSE.
+
+
+
+== Install Uyuni Proxy on openSUSE Leap 15.1
```

Comment:
IIRC the version was now a variable, so we don't need to do a replace on all documents when we change to `15.2`

Reply:
[I think it would be even better, if we could avoid the version number here and at several other places completely.  And mention the version number somewhere in the intro.]

Unfortunately, I do not no where you must define those variables.

Reply:
Maybe they are already defined. There are files at the root folder defining the stuff such as `{productname}`, etc. Check with the team if you have issues understanding how it works.

Reply:
ok, will check with the team.